### ISSUE TITLE
Print intro text on game startup

### ISSUE THREAD

**Issue by borjasotomayor** [uchicago-cs/chiventure#404]
When a game starts, the "intro" text (in the "GAME" portion of the WDL file) should be printed out. For example, if we run chiventure with the `distaff.yaml` game, the opening screen should look something like this:

```
┌─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│              ________________________________________________________________________________________               │
│            / \                                                                                       \              │
│           |   |                                                                                      |              │
│            \_ |     ██████╗██╗  ██╗██╗██╗   ██╗███████╗███╗   ██╗████████╗██╗   ██╗██████╗ ███████╗  |              │
│               |    ██╔════╝██║  ██║██║██║   ██║██╔════╝████╗  ██║╚══██╔══╝██║   ██║██╔══██╗██╔════╝  |              │
│               |    ██║     ███████║██║██║   ██║█████╗  ██╔██╗ ██║   ██║   ██║   ██║██████╔╝█████╗    |              │
│               |    ██║     ██╔══██║██║╚██╗ ██╔╝██╔══╝  ██║╚██╗██║   ██║   ██║   ██║██╔══██╗██╔══╝    |              │
│               |    ╚██████╗██║  ██║██║ ╚████╔╝ ███████╗██║ ╚████║   ██║   ╚██████╔╝██║  ██║███████╗  |              │
│               |     ╚═════╝╚═╝  ╚═╝╚═╝  ╚═══╝  ╚══════╝╚═╝  ╚═══╝   ╚═╝    ╚═════╝ ╚═╝  ╚═╝╚══════╝  |              │
│               |   ___________________________________________________________________________________|___           │
│               |  /                                                                                      /           │
│               \_/______________________________________________________________________________________/            │
│                                                                                                                     │
│                                                                                                                     │
│                                            Type 'HELP' to show help menu                                            │
└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘

Welcome adventurer! You have made your way to the dungeons of the Lavender Palace. Don't be fooled by the cute name! 
This is one of Throxar the Terrible's most fearsome strongholds... and also his best scented one. Queen Vanadia has
tasked you with recovering the Distaff of Knowledge, which her spies assure you is somewhere in these dungeons.

  > 
```

When chiventure is run without a game, there should be some sort of message to that effect. For example:

```
┌─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│              ________________________________________________________________________________________               │
│            / \                                                                                       \              │
│           |   |                                                                                      |              │
│            \_ |     ██████╗██╗  ██╗██╗██╗   ██╗███████╗███╗   ██╗████████╗██╗   ██╗██████╗ ███████╗  |              │
│               |    ██╔════╝██║  ██║██║██║   ██║██╔════╝████╗  ██║╚══██╔══╝██║   ██║██╔══██╗██╔════╝  |              │
│               |    ██║     ███████║██║██║   ██║█████╗  ██╔██╗ ██║   ██║   ██║   ██║██████╔╝█████╗    |              │
│               |    ██║     ██╔══██║██║╚██╗ ██╔╝██╔══╝  ██║╚██╗██║   ██║   ██║   ██║██╔══██╗██╔══╝    |              │
│               |    ╚██████╗██║  ██║██║ ╚████╔╝ ███████╗██║ ╚████║   ██║   ╚██████╔╝██║  ██║███████╗  |              │
│               |     ╚═════╝╚═╝  ╚═╝╚═╝  ╚═══╝  ╚══════╝╚═╝  ╚═══╝   ╚═╝    ╚═════╝ ╚═╝  ╚═╝╚══════╝  |              │
│               |   ___________________________________________________________________________________|___           │
│               |  /                                                                                      /           │
│               \_/______________________________________________________________________________________/            │
│                                                                                                                     │
│                                                                                                                     │
│                                            Type 'HELP' to show help menu                                            │
└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘

No game has been loaded! Use the LOAD command to load a game.

  > 
```

**Comment by ecoble** [comment]
Game now prints the intro text or no game loaded message on start up

**Comment by borjasotomayor** [comment]
Closed by #407
